### DISCUSSION TITLE
7月5号购买1年账号，请尽快提供激活码

### DISCUSSION THREAD

**Discussion by Z199-Hai** [getlantern/lantern-forum-cn#703] [General]
@Mike777Khoo 
账号邮箱：2985603424@qq.com
付款截图如下，请提供激活码，谢谢！
![买蓝灯账号2 340支付截图](https://github.com/getlantern/lantern-forum-cn/assets/173046878/58ba15e8-b87d-4e26-980c-efca9668b60d)

**Comment by MaxMKMax** [comment]
@Z199-Hai 您好. 我们会帮您解决问题. 请您放心. 为了保护您的隐私, 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱. 
后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您!
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ

*备注: 如果各位看到这个链接, 请您依然开个帖子说明您的问题. 这样我们才能发现到您的问题帮您跟进. 不要只点连接提交问题而不开帖子. 帖子上不需要个人资料了. 邮箱和支付单据放在链接里就好了. 谢谢!

**Reply by Z199-Hai** [reply]
已经提交资料了，请尽快提供激活码，谢谢

**Reply by MaxMKMax** [reply]
@Z199-Hai 您好. 请问你收到激活码了吗?
